packages/cyberstorm/src/components/componentStyles/MenuItem.module.css

```diff
@@ -18,25 +18,29 @@
   --bg-color: transparent;
 }
 
+.root[data-highlighted] {
+  z-index: 999;
+}
+
 .label {
   font-weight: var(--font-weight-500);
   line-height: var(--line-height-l);
 }
 
 /* Different MenuItem color schemes */
-.selectItem__default:hover {
+.selectItem__default[data-highlighted] {
```

Comment:
I don't think this change is necessary, since we specifically want the style changes on hover, not when another component might or might not add a data attribute.

This applies to the changes below as well.

Reply:
Reverted these.